go.mod

```diff
@@ -41,8 +41,8 @@ require (
 	github.com/rancher/dynamiclistener v0.3.5
 	github.com/rancher/lasso v0.0.0-20230830164424-d684fdeb6f29
 	github.com/rancher/lasso/controller-runtime v0.0.0-20230830164424-d684fdeb6f29
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240507200315-e6403cd399d7
-	github.com/rancher/rke v1.4.18-rc1
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240610223012-5e307170c82f
```

Comment:
This may need to be bumped to a commit of https://github.com/rancher/rancher/pull/45851